feature/resume/impl/src/main/java/com/example/impl/step4/ResumeStep4Screen.kt

```
@@ -0,0 +1,4 @@
+package com.example.feature.resume.impl.step4
+
+class ResumeStep4Screen {
+}
```

detekt (1.23.8)

[warning] The class or object ResumeStep4Screen is empty.

(detekt.empty-blocks.EmptyClassBlock)